xml/admin_operating_pools.xml

```diff
@@ -890,23 +890,47 @@ pool4                     1.1 MiB      13      0     39                  0
    need to migrate the pool data to another one whose parameters suit your
    deployment.
   </para>
-
   <para>
-   There are several methods of pool migration. We recommend using
-   <emphasis>cache tier</emphasis> because this method is transparent, reduces
-   cluster downtime, and avoids duplicating the entire pool's data.
+   There are two migration methods&mdash;a <emphasis>cache tier</emphasis>
+   method for general pool data migration, and a method using <command>rbd
+    migrate</command> sub-commands to migrate RBD images to a new pool. Each
+   method has its specifics and limitations.
   </para>
 
+  <sect2 xml:id="pool-migrate-limits">
+   <title>Limitations</title>
+   <itemizedlist>
+    <listitem>
+     <para>
+      You can use the cache tier method to migrate from a replicated pool to
```

Comment:
Should this not be highlighted/emphasized  as the `rbd migration` method is, below?